daemon/daemon_solaris.go

```diff
@@ -84,6 +320,16 @@ func (daemon *Daemon) platformReload(config *Config) map[string]string {
 
 // verifyDaemonSettings performs validation of daemon config struct
 func verifyDaemonSettings(config *Config) error {
+
+	if config.DefaultRuntime == "" {
+		config.DefaultRuntime = stockRuntimeName
```

Comment:
Microsoft folks had an issue with that. stockRuntimeName would have to be defined in config_windows as well and I was brought to understand they don't really have a stock runtime..
We've gotten around this by creating a symlink to runz as runc. So the current logic is fine.